### PR TITLE
Upgrade scalafmt to 2.0.0-RC7

### PR DESCRIPTION
**Pull request by chbatey**
Not many changes, most of which look good to me

### CODE REVIEW

File: akka-cluster-metrics/src/main/scala/akka/cluster/metrics/Metric.scala [hunk 1]
```diff
@@ -183,14 +182,13 @@ object StandardMetrics {
...
         processors <- nodeMetrics.metric(Processors)
-      } yield
-        (
-          nodeMetrics.address,
-          nodeMetrics.timestamp,
-          nodeMetrics.metric(SystemLoadAverage).map(_.smoothValue),
-          nodeMetrics.metric(CpuCombined).map(_.smoothValue),
-          nodeMetrics.metric(CpuStolen).map(_.smoothValue),
-          processors.value.intValue)
+      } yield (
```
Comment (chbatey) [inline akka-cluster-metrics/src/main/scala/akka/cluster/metrics/Metric.scala]: Much nicer

File: akka-cluster-metrics/src/main/scala/akka/cluster/metrics/Metric.scala [hunk 2]
```diff
@@ -131,13 +131,12 @@ object StandardMetrics {
...
         used <- nodeMetrics.metric(HeapMemoryUsed)
         committed <- nodeMetrics.metric(HeapMemoryCommitted)
-      } yield
-        (
-          nodeMetrics.address,
-          nodeMetrics.timestamp,
-          used.smoothValue.longValue,
-          committed.smoothValue.longValue,
-          nodeMetrics.metric(HeapMemoryMax).map(_.smoothValue.longValue))
+      } yield (
```
Comment (chbatey) [inline akka-cluster-metrics/src/main/scala/akka/cluster/metrics/Metric.scala]: Much nicer

File: akka-persistence/src/test/scala/akka/persistence/PersistenceSpec.scala [hunk 3]
```diff
@@ -117,11 +117,10 @@ trait PersistenceMatchers {
         nrs = seq.map(_.replaceFirst(prefixes(pos), "").toInt)
         sortedNrs = nrs.sorted
         if nrs != sortedNrs
-      } yield
-        MatchResult(
-          false,
-          s"""Messages sequence with prefix ${prefixes(pos)} was not sorted! Was: $seq"""",
-          s"""Messages sequence with prefix ${prefixes(pos)} was sorted! Was: $seq"""")
+      } yield MatchResult(
```
Comment (chbatey) [inline akka-persistence/src/test/scala/akka/persistence/PersistenceSpec.scala]: :+1:

File: akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerMTSpec.scala [hunk 4]
```diff
@@ -37,18 +37,17 @@ class CircuitBreakerMTSpec extends AkkaSpec {
     def testCallsWithBreaker(): immutable.IndexedSeq[Future[String]] = {
       val aFewActive = new TestLatch(5)
       for (_ <- 1 to numberOfTestCalls)
-        yield
-          breaker
-            .withCircuitBreaker(Future {
+        yield breaker
```
Comment (chbatey) [inline akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerMTSpec.scala]: :+1:

File: akka-actor-tests/src/test/scala/akka/actor/SchedulerSpec.scala [hunk 5]
```diff
@@ -298,11 +298,10 @@ trait SchedulerSpec extends BeforeAndAfterEach with DefaultTimeout with Implicit
       }
       val latencies = within(10.seconds) {
         for (i <- 1 to N)
-          yield
-            try expectMsgType[Long]
-            catch {
-              case NonFatal(e) => throw new Exception(s"failed expecting the $i-th latency", e)
-            }
+          yield try expectMsgType[Long]
```
Comment (chbatey) [inline akka-actor-tests/src/test/scala/akka/actor/SchedulerSpec.scala]: :+1:

**Review by chbatey — COMMENTED** [review]
All good imo

**Review by raboof — APPROVED** [review]
Seems a bit nicer, and not too many changes so not too many conflicts

**Review by patriknw — APPROVED** [review]
LGTM since it's not changing much, otherwise we should wait until final unless something is broken (trouble for open PRs)

**Comment by patriknw** [comment]
I'm tempted to backport this to keep the format in sync but let's see if we run into any problems first (hopefully the final isn't too far out)

**Pull request merged**